core/domain/src/test/kotlin/com/google/samples/apps/nowinandroid/core/domain/GetRecentSearchQueriesUseCaseTest.kt

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright 2024 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.samples.apps.nowinandroid.core.domain
+
+import com.google.samples.apps.nowinandroid.core.testing.repository.TestRecentSearchRepository
+import com.google.samples.apps.nowinandroid.core.testing.util.MainDispatcherRule
+import kotlinx.coroutines.delay
+import kotlinx.coroutines.flow.first
+import kotlinx.coroutines.test.runTest
+import org.junit.Rule
+import org.junit.Test
+import kotlin.test.assertEquals
+
+class GetRecentSearchQueriesUseCaseTest {
+
+    @get:Rule
+    val mainDispatcherRule = MainDispatcherRule()
+
+    private val recentSearchRepository = TestRecentSearchRepository()
+
+    private val useCase = GetRecentSearchQueriesUseCase(
+        recentSearchRepository,
+    )
+
+    @Test
+    fun whenNoParams_recentSearchQueriesAreReturnedUpTo10() = runTest {
```

Comment:
Params should be replaced by its actual name, which is "limit"

Reply:
yes, your opinion is more reasonable. but, test code naming convention(='whenNoParams_followableTopicsAreReturnedWithNoSorting' in 'GetFollowableTopicsUseCaseTest') is not correct.

so according to what you said, is it correct that this method name should also be modified? if yes, i will try one of them.

1. Modifying the method name of 'whenNoParams_followableTopicsAreReturnedWithNoSorting' together and pull request.
2. Register as another issue and proceed.